src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:15981/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:15981/project/issues?id=sample&branch=feature1&issues=AWEI7_5aG51Al2-FAh-0&open=AWEI7_5aG51Al2-FAh-0)

[ISSUE_KEY]: AWEI7_5aG51Al2-FAh-0